templates/index.html

```diff
@@ -50,450 +50,482 @@
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="http://github.com/fossasia/meilix-generator/issues" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='bug.png') }}"></div>
+              <p class="custom-title">Bug Report</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="http://blog.fossasia.org" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='blog.png') }}"></div>
+              <p class="custom-title">Blogs</p>
+            </a>
+          </div>
+          <hr style="margin-bottom: 10px; margin-top: 10px;">
+          <div class="custom-menu-item">
+            <a href="https://eventyay.com/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='eventyay.png') }}"></div>
+              <p class="custom-title">Eventyay</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://susper.com/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='susper.png') }}" style="width: 60px;height: 16px;"></div>
+              <p class="custom-title">Susper</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://chat.susi.ai/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='susi.png') }}"></div>
+              <p class="custom-title">Susi</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://fossasia.org/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='fossasia.png') }}"></div>
+              <p class="custom-title">FOSSASIA</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://loklak.org/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='loklak.png') }}"></div>
+              <p class="custom-title">loklak</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://phimp.me/" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='phimp.png') }}"></div>
+              <p class="custom-title">Phimp.me</p>
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://pslab.fossasia.org" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='Pslab.png') }}"></div>
+              <p class="custom-title">PS Lab</p>
+            </a>
+          </div>
+          <hr style="margin: 10px">
+          <div style="display: flex;justify-content: center;align-items: center; margin: 0 0 -20px 0">
```

Comment:
Is it really necessary to put style in the html template? try to use CSS for MVC compliance.